docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3"
+
+services:
+  app:
+    image: digital-collections
+    build:
+      context: .
+      args:
+        - NODE_ENV=local
```

Comment:
This should be `NODE_ENV=production`.

Reply:
this points to the Dockerfile that has its own NODE_ENV value set but it's currently commented out. It's fine to set it in both.